### DISCUSSION TITLE
movie character model in russian language?

### DISCUSSION THREAD

**Discussion by Dmitrydavka** [coqui-ai/TTS#2901] [General Q&A] [answered]
Hello guys, I need to make text-to-speech with the voice of one movie character in russian language. Using svc so-vits, I was able to get a good model from an hour of conversations, voice cloning works fine for some songs, but I couldn't make tts - as i get it, it does not support Russian. Will I able to train the character model and make text-to-speech using coqui-ai in russian?

**Comment by erogol** [comment, marked as answer]
yes you can train models for any language using TTS

There should also be a Russian model 

https://tts.readthedocs.io/en/latest/inference.html#example-text-to-speech-using-fairseq-models-in-1100-languages

**Reply by Dmitrydavka** [reply]
Thank you, i will check it!

**Comment by anni-anni** [comment]
You can try training a [text to speech AI](https://wavel.ai/solutions/realistic-text-to-speech-online/) that supports Russian, has a good voice library and relevant accents. The AI that I use for multilingual [voice cloning](https://wavel.ai/studio/ai-voice-cloner/) has some great features and supports Russian too
